ci/kind/kind-setup.sh

```diff
@@ -232,8 +236,12 @@ function configure_extra_networks {
   networks=()
```

Comment:
I don't think this function is related to0 hybrid mode?

Reply:
Removed the changes. Additionally, I found that it seems the option `--extra-network` of `ci/kind/test-e2e-kind.sh` introduced in #5416 is no longer used in current KinD tests. cc @jianjuns